.github/scripts/run_consensus.sh

```diff
@@ -8,7 +8,7 @@ mkdir -p docker/data/
 source docker/env
 
 # Generate chainspec and populate comittee keystores
-docker run -v $(pwd)/docker/data:/data --entrypoint "/bin/sh" -e DAMIAN -e TOMASZ -e ZBYSZKO -e HANSU -e RUST_LOG=info aleph-node:latest -c "aleph-node bootstrap-chain --base-path /data --chain-id a0dnet1 --millisecs-per-block 2000 --session-period 10 --account-ids $DAMIAN,$TOMASZ,$ZBYSZKO,$HANSU --sudo-account-id $DAMIAN > /data/chainspec.json"
+docker run -v $(pwd)/docker/data:/data --entrypoint "/bin/sh" -e DAMIAN -e TOMASZ -e ZBYSZKO -e HANSU -e RUST_LOG=info aleph-node:latest -c "aleph-node bootstrap-chain --raw --base-path /data --chain-id a0dnet1 --millisecs-per-block 2000 --session-period 10 --account-ids $DAMIAN,$TOMASZ,$ZBYSZKO,$HANSU --sudo-account-id $DAMIAN > /data/chainspec.json"
```

Comment:
do we need raw chainspec in pipelines now?

Reply:
not neccessarily, no - it will work with both